_pages/calls/industrytrack.md

```diff
@@ -0,0 +1,113 @@
+---
+title:
+layout: single
+permalink: /_pages/calls/
```

Comment:
The permalink should just be `/calls/industry/` since that's what the URL will be.